doc/classes/EditorSettings.xml

```diff
@@ -877,6 +880,17 @@
 		<member name="text_editor/completion/use_single_quotes" type="bool" setter="" getter="">
 			If [code]true[/code], performs string autocompletion with single quotes. If [code]false[/code], performs string autocompletion with double quotes (which matches the [url=$DOCS_URL/tutorials/scripting/gdscript/gdscript_styleguide.html]GDScript style guide[/url]).
 		</member>
+		<member name="text_editor/external/exec_flags" type="String" setter="" getter="">
+			Arguments passed to the external test editor on execution.
+		</member>
+		<member name="text_editor/external/exec_path" type="String" setter="" getter="">
+			File path of the external text editor binary.
+		</member>
+		<member name="text_editor/external/use_external_editor" type="bool" setter="" getter="">
+			If [code]true[/code], an external text editor will be used for editing script files.
+			[b]Note:[/b] If setting an external editor for C#, set [member dotnet/editor/external_editor] instead.
```

Comment:
While it's true that C# users may prefer to set `dotnet/editor/external_editor`, I feel like this makes it sound as if using this setting is wrong but that shouldn't be the case.